packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Fixed
+- SF-1237: Fix issue where past purchase sorts are not correctly applied/transmitted.
+
```

Comment:
```
### Added
- SF-1237: Past purchase sort configuration
  - `PAST_PURCHASE_SORTS` exported
    - `DEFAULT` sorts past purchases based on the past purchase endpoint
    - `MOST_PURCHASED` sorts past purchases based on the quantity purchased
    - `MOST_RECENT` sorts past purchases based on the latest purchased timestamp
  - eg, {
      recommendations: {
        pastPurchases: {
          sort: {
            options: [{ field: PAST_PURCHASE_SORTS.MOST_RECENT, descending: true }, { field: PAST_PURHCASE_SORTS. MOST_PURCHASED, descending: true }],
            default: { field: PAST_PURCHASE_SORTS.MOST_RECENT, descending: true }
        }
      }
    }
```
p sure that's correct.

Reply:
youcan fixed the words some

Reply:
But make sure to actually format the object as a code block if you're going to include it.

Reply:
Also, CRAFDS, so make sure to put `Added` before `Fixed`.

Reply:
Implemented.